src/js/components/inspectors/FormInputProperty.jsx

```diff
@@ -186,6 +188,48 @@ var FormInputProperty = React.createClass({
           </select>
         );
 
+      case 'toggle':
+        var currentVal = sg.get(props.signal);
+        for (var i = 0; i < props.opts.length; i++) {
+          var val = props.opts[i];
```

Comment:
Careful with `var` declarations within loops or if statements. JavaScript does not have block-level variable scoping. Instead, with Lyra, we prefer declaring all the variables a function uses at the very top. If you execute `npm run lint`, the linter should pick up this (and other) issues.